charts/otomi-api/templates/istio/authorizationpolicy.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: security.istio.io/v1
```

Comment:
Why do we need AuthorizationPolicy if apl-api validates authorization itself?

Reply:
apl-api does not validate the authorization, it only checks for a well-formatted token and if it contains a `user` field. The RequestAuthentication by itself did not have the desired effect of providing the signature check. Apparently the rule in the AuthorizationPolicy is needed to make this happen.

Reply:
Possibly RequestAuthorization does not even decode the token unless something from its content, e.g. `requestPrincipals` is being checked.